classes/fields/datetime.php

```diff
@@ -718,6 +722,33 @@ public function createFromFormat( $format, $date, $return_timestamp = false ) {
 		return apply_filters( 'pods_form_ui_field_datetime_formatter', $datetime, $format, $date );
 	}
 
+	/**
+	 * Check if a value is compatible with the storage format.
+	 *
+	 * @param  string $value
+	 * @return bool
+	 */
+	public function is_storage_format( $value ) {
+		$value_parts  = str_split( $value );
+		$format_parts = str_split( date( static::$storage_format ) );
```

Comment:
:no_entry_sign: date() is affected by runtime timezone changes which can cause date/time to be incorrectly displayed. Use gmdate() instead.

<sub>hash: 58163224d56cd1f22fc714c2911b0765</sub>